media/ivf.ksy

```diff
@@ -0,0 +1,67 @@
+meta:
+  id: ivf
```

Comment:
Formats repo have global IDs without any namespacing. Given that "ivf" is not a very popular format and there might be other formats identified as "ivf", may I suggest to rename this (and the file) to something more verbose, like `vp8_ivf`, as this is clearly a container for VP8?